tenderly.yaml

```
@@ -1,6 +1,6 @@
 account_id: ""
 actions:
-  Swapr/swapr:
+  berteotti/project:
```

We don't want this change

```suggestion
  Swapr/swapr:
```

---

fixed